build.gradle

```diff
@@ -7,31 +7,23 @@ buildscript {
             name = "forge"
             url = "http://files.minecraftforge.net/maven"
         }
-        maven { url "https://plugins.gradle.org/m2/" }
     }
     dependencies {
         classpath 'net.minecraftforge.gradle:ForgeGradle:2.1-SNAPSHOT'
     }
 }
+
 apply plugin: 'net.minecraftforge.gradle.tweaker-client'
 apply plugin: 'java'
 
-/*
-// for people who want stable - not yet functional for MC 1.8.8 - we require the forgegradle 2.1 snapshot
-plugins {
-    id "net.minecraftforge.gradle.forge" version "2.0.2"
-}
-*/
-
 jar {
-    // Add the license to the jar
     from "LICENSE"
 }
 
 targetCompatibility = sourceCompatibility = 1.8
 
-version = "2.0.3"
-group= "co.bugg.quickplay" // http://maven.apache.org/guides/mini/guide-naming-conventions.html
+version = "2.0.4-asbyth"
```

Comment:
Will need to be changed to 2.1.0

Reply:
unfortunately dont have the repository anymore, would have to be changed after merging